### ISSUE TITLE
[Accessibility Audit] Frontend Platform: Branches page

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34424]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1rSp77P0UDY6ewHq6iXmW0nuilBtkMW05n1ciMB8gQ3w/edit?usp=sharing). Use this for further context.

Note: This page doesn’t have significant changes after the repository redesign. Double check this is still the case before actioning these user journeys in any way.

View branches - Overview:

- Go to branches tab
- View the default branch
- View active branches
- Click on a branch to navigate back to the repository tree page with the selected branch used as the repository revision.
- Click “View more branches” to navigate to the “All branches” tab (See below)

View branches - All branches:

- Go to branches tab
- Click “All branches”
- Click on a branch to navigate back to the repository tree page with the selected branch used as the repository revision.
- Focus the input and enter text to search for specific branches
- Select “Show more” to load more branches

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by umpox** [comment]
Audit complete. Linked issues attached.
